initramfs-init.in

```diff
@@ -646,6 +652,22 @@ if [ -n "$ovlfiles" ]; then
 else
 	apk add --root $sysroot $repo_opt $apkflags $pkgs
 fi
+
+if [ -n "$ssh_install" ]; then
+	rc_add sshd default
+	echo "PermitRootLogin yes" >> $sysroot/etc/ssh/sshd_config
+	if [ -n "$ssh_key" ]; then
+		mkdir -p $sysroot/root/.ssh
+		wget "$ssh_key" -O $sysroot/root/.ssh/authorized_keys
```

Comment:
How about:
````sh
￼if [ -n "$ssh_key" ]; then
￼    mkdir -p $sysroot/root/.ssh
    case "$ssh_key" in
        http://*|https://*) wget "$ssh_key" -O $sysroot/root/.ssh/authorized_keys;;
        *) echo "$ssh_key" > "$sysroot"/root/.ssh/authorized_keys;;
    esac
fi
````

That way you can also paste in your ed25519 key in the boot option.

Reply:
That's nice. Fixed. Thanks.